Sources/BSON/Date+BSONValue.swift

```diff
@@ -46,6 +46,20 @@ extension Date: BSONValue {
         }
     }
 
+    /// Converts this `BSONDate` to a corresponding `JSON` in relaxed extendedJSON format.
+    internal func toRelaxedExtendedJSON() -> JSON {
+        if self.msSinceEpoch >= 0 && self.msSinceEpoch <= 253_402_318_800 {
```

Comment:
can we add a comment here indicating where this 253... number came from for future readers?

Reply:
yes